Library/Formula/cpp-netlib.rb

```diff
@@ -15,6 +21,8 @@ class CppNetlib < Formula
 
   depends_on "cmake" => :build
   depends_on "openssl"
+  # Version 0.12.0+ move from boost::asio to asio.
+  depends_on "asio"  if build.devel?
```

Comment:
This should be in the `devel do` block FYI, not tagged with `if build.devel?` here.